cuda/components/prefix_sum.cu

```diff
@@ -30,10 +30,10 @@ THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
 OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 ******************************<GINKGO LICENSE>*******************************/
 
-#include "core/components/prefix_sum.hpp"
+#include "cuda/components/prefix_sum.cuh"
 
 
-#include "cuda/components/prefix_sum.cuh"
+#include "core/components/prefix_sum.hpp"
```

Comment:
I would rather leave the previous order, as this is the implementation of the prefix_sum kernel declared in `core/components/prefix_sum.hpp`. Does this have to do with the fact that the file name does not end with `kernels`?